lib/local-links-manager/check_links/homepage_status_updater.rb

```diff
@@ -21,7 +21,7 @@ def update
     private
 
       def links
-        table.distinct.pluck(column)
+        table.distinct.where.not(column => '').pluck(column)
```

Comment:
We thought there might be a nice Active Record method that would handle both nil and empty strings, but this was the neatest solution we found :)